features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1587,6 +1595,22 @@
             <Permissions>/permission/admin/manage/identity/rolemgt/delete</Permissions>
             <Scopes>internal_role_mgt_delete</Scopes>
         </Resource>
+        <Resource context="(.*)/scim2/Roles/(.*)" secured="true" http-method="GET">
```

Comment:
Why not have all the "(.*)/scim2/Roles/(.*)" entries together?

Reply:
Groups config had the same way. Moved to one place and updated the PR.